operate/client/e2e-playwright/mocks/decisionInstance.mocks.ts

```diff
@@ -76,163 +77,7 @@ const mockEvaluatedDrdData: DrdDataDto = {
   ],
 };
 
-const mockEvaluatedXml = `<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These XML strings were moved to e2e-playwright/mocks/resources as dmn xml files without changes